lm/util.py

```diff
@@ -0,0 +1,24 @@
+from typing import List, Tuple
+
+import i6_core.rasr as rasr
+
+
+def _has_image(c: rasr.RasrConfig, pc: rasr.RasrConfig):
+    res = c._get("image") is not None
+    res = res or (pc is not None and pc._get("image") is not None)
+    return res
+
+
+def find_arpa_lms(lm_config: rasr.RasrConfig, lm_post_config=None) -> List[Tuple[rasr.RasrConfig, rasr.RasrConfig]]:
+    result = []
+
+    if lm_config.type == "ARPA":
+        if not _has_image(lm_config, lm_post_config):
```

Comment:
Is the not correct here? So if there is no image it adds it to result?

Reply:
I think the idea is not to overwrite existing LM images.